src/pages/Main/Main.js

```diff
@@ -1,13 +1,69 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { withRouter } from 'react-router-dom';
 import styled from 'styled-components';
-
+import axios from 'axios';
+import BtnImage from './Btn';
 function Main() {
+  const [User, setUser] = useState({
+    productList: [],
+  });
+  useEffect(() => {
+    axios.get('/data/User.json').then(response => {
+      setUser({ productList: response.data });
```

Comment:
- 그렇다면 state 를 업데이트 할때 다음과 같은 과정으로 될 것 같습니다.
```suggestion
      setProductList(response.data);
```